images/protected-publish/validate_index.py

```diff
@@ -0,0 +1,120 @@
+import argparse
+import io
+import json
+import re
+import sys
+from datetime import datetime
+
+import botocore.exceptions
+import boto3.session
+
+
```

Comment:
Should we also include sentry in this file? Or, is this never run in the cluster?

Reply:
I'll add it back, since as you pointed out, it doesn't hurt to have it in there, even if there is no cronjob defined to run it.